hack/update-gofmt.sh

```diff
@@ -16,6 +16,9 @@
 
 # GoFmt apparently is changing @ head...
 
+# This script runs GoFmt to golang source codes
```

Comment:
GoFmt
advice:
`go fmt`

Reply:
Thanks, your advice seems right.
I will fix it.